sky/clouds/service_catalog/__init__.py

```diff
@@ -324,13 +324,13 @@ def get_common_gpus() -> List[str]:
         'A100',
         'A100-80GB',
         'H100',
-        'K80',
         'L4',
-        'M60',
+        'L40S',
         'P100',
         'T4',
         'V100',
         'V100-32GB',
+        'MI25'
```

Comment:
This isn't "common"; avoid?

Reply:
This is the only AMD GPU that might be good to show.

Reply:
@concretevitamin wdyt?

Reply:
I'd propose skipping.